packages/vm/src/buildBlock.ts

```diff
@@ -85,7 +85,11 @@ export class BlockBuilder {
       this.vm.common.isActivatedEIP(1559) === true &&
       typeof this.headerData.baseFeePerGas === 'undefined'
     ) {
-      this.headerData.baseFeePerGas = opts.parentBlock.header.calcNextBaseFee()
+      if (this.headerData.number === vm.common.hardforkBlock(Hardfork.London)) {
+        this.headerData.baseFeePerGas = vm.common.param('gasConfig', 'initialBaseFee')
```

Comment:
This could also be moved back into client, but I think the place for this change is better here?

Note: it doesn't seem that we have access to the parent block in this scope, so cannot check parent gas limit here... (?)

Reply:
`opts.parentBlock` could be available here

Reply:
added a commit to address this